### ISSUE TITLE
[Bug]: kotlin.UninitializedPropertyAccessException in IntelliJ 2023.2.3

### ISSUE THREAD

**Issue by Adriansun** [mallowigi/material-theme-issues#415]
### Describe the bug

Thank you for the quick update for the IntelliJ 2023.2.3 version. Went great at work to update Material Theme UI. Then I went home and did the same and got:

kotlin.UninitializedPropertyAccessException: lateinit property DESCRIPTORS has not been initialized
	at com.mallowigi.idea.config.MTFileColorsPage.getColorDescriptors(MTFileColorsPage.kt:58)
	at com.intellij.application.options.colors.ColorAndFontOptions.initDescriptions(ColorAndFontOptions.java:586)
	at com.intellij.application.options.colors.ColorAndFontOptions.initPluggedDescriptions(ColorAndFontOptions.java:542)
	at com.intellij.application.options.colors.ColorAndFontOptions.initScheme(ColorAndFontOptions.java:531)
	at com.intellij.application.options.colors.ColorAndFontOptions.initAll(ColorAndFontOptions.java:515)
	at com.intellij.application.options.colors.ColorAndFontOptions.resetImpl(ColorAndFontOptions.java:656)
	at com.intellij.application.options.colors.ColorAndFontOptions.reset(ColorAndFontOptions.java:668)
	at com.mallowigi.idea.MTThemeManager.resetColorScheme(MTThemeManager.kt:872)
	at com.mallowigi.idea.MTLafComponent.appStarted$lambda$1(MTLafComponent.kt:77)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:208)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:190)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.openapi.application.impl.ApplicationImpl$4.run(ApplicationImpl.java:478)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:100)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:79)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:121)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:41)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:792)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:690)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:593)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWithoutImplicitRead(ApplicationImpl.java:1485)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:593)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:997)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:997)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

### To Reproduce

Update everything to the latest.

### Expected behavior

Not to have a Kotlin crash at the IntelliJ project start up window.

### Material Theme Version

8.12.2

### IDE

IntelliJ IDEA

### System Info

```shell
Windows 10 Latest
Material Theme UI 8.12.2
IntelliJ 2023.2.3
```

### Logs/Screenshots

```shell
kotlin.UninitializedPropertyAccessException: lateinit property DESCRIPTORS has not been initialized
	at com.mallowigi.idea.config.MTFileColorsPage.getColorDescriptors(MTFileColorsPage.kt:58)
	at com.intellij.application.options.colors.ColorAndFontOptions.initDescriptions(ColorAndFontOptions.java:586)
	at com.intellij.application.options.colors.ColorAndFontOptions.initPluggedDescriptions(ColorAndFontOptions.java:542)
	at com.intellij.application.options.colors.ColorAndFontOptions.initScheme(ColorAndFontOptions.java:531)
	at com.intellij.application.options.colors.ColorAndFontOptions.initAll(ColorAndFontOptions.java:515)
	at com.intellij.application.options.colors.ColorAndFontOptions.resetImpl(ColorAndFontOptions.java:656)
	at com.intellij.application.options.colors.ColorAndFontOptions.reset(ColorAndFontOptions.java:668)
	at com.mallowigi.idea.MTThemeManager.resetColorScheme(MTThemeManager.kt:872)
	at com.mallowigi.idea.MTLafComponent.appStarted$lambda$1(MTLafComponent.kt:77)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:208)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:190)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.openapi.application.impl.ApplicationImpl$4.run(ApplicationImpl.java:478)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:100)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:79)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:121)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:41)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:792)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:690)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:593)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWithoutImplicitRead(ApplicationImpl.java:1485)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:593)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:997)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:997)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

**Comment by mallowigi** [comment]
Duplicate. I'm aware of this issue but unfortunately I don't have the time to take care of it as my country is currently at war. Will do whenever I have the time.

**Comment by david-idelji** [comment]
> Duplicate. I'm aware of this issue but unfortunately I don't have the time to take care of it as my country is currently at war. Will do whenever I have the time.

Sorry for your situation and please take care and be safe!

**Comment by mallowigi** [comment]
Should be fixed on 8.13.0

**Comment by BlockyTheDev** [comment]
> Should be fixed on 8.13.0

Thanks for the fix. Stay safe and take care!
